package.json

```diff
@@ -1101,15 +1101,16 @@
     "ajv": "^4.7.5",
     "es6-promisify": "~5.0.0",
     "follow-redirects": "^1.5.10",
+    "handlebars": "^4.5.3",
```

Comment:
Is this needed? I see it in the `resolutions` section already and we don't actively use this in the extension.

Reply:
You can also remove `follow-redirects`. We forgot to remove that in a previous commit.